src/components/Reactions/types.ts

```diff
@@ -3,10 +3,13 @@ import type { ReactionResponse } from 'stream-chat';
 
 export interface ReactionSummary {
   EmojiComponent: ComponentType | null;
+  firstReactionAt: Date | null;
   isOwnReaction: boolean;
+  lastReactionAt: Date | null;
   latestReactedUserNames: string[];
   reactionCount: number;
   reactionType: string;
+  unlistedReactedUserCount: number;
```

Comment:
Just useful in case someone wants to implement "John, Jane, and 5 others" style tooltips.